data_steward/cdr_cleaner/cleaning_rules/fitbit_dateshift.py

```diff
@@ -0,0 +1,234 @@
+"""
+Date shifting fitbit tables.
+
+Extends the basic date shifting rule by providing table names
+and schemas.
+
+Original Issue:  DC-1005
+"""
+# Python Imports
+import logging
+
+# Project imports
+from cdr_cleaner.cleaning_rules.dateshift import DateShiftRule
+from constants.cdr_cleaner import clean_cdr as cdr_consts
+from resources import fields_for
+
+LOGGER = logging.getLogger(__name__)
+
+ISSUE_NUMBERS = ['DC-1005']
+
+
+class FitbitDateShiftRule(DateShiftRule):
+    """
+    Date shift any date, datetime, or timestamp fields in the fitbit tables.
+    """
+
+    def __init__(self, project_id, dataset_id, sandbox_dataset_id,
+                 map_dataset_id, map_tablename):
+        """
+        Initialize the class.
+
+        Set the issue numbers, description and affected datasets.  As other
+        tickets may affect this SQL, append them to the list of Jira Issues.
+        DO NOT REMOVE ORIGINAL JIRA ISSUE NUMBERS!
+        """
+        desc = (f'Date shift date and timestamp fields by the date shift '
+                f'calculated in the static mapping table.  This specifically '
+                f'applies to fitbit tables.')
+
+        self.tables = [
+            'activity_summary', 'heart_rate_minute_level', 'heart_rate_summary',
+            'steps_intraday'
+        ]
+        super().__init__(issue_numbers=ISSUE_NUMBERS,
+                         description=desc,
+                         affected_datasets=[cdr_consts.FITBIT],
+                         project_id=project_id,
+                         dataset_id=dataset_id,
+                         sandbox_dataset_id=sandbox_dataset_id,
+                         affected_tables=self.tables,
+                         map_dataset=map_dataset_id,
+                         map_table=map_tablename,
+                         depends_on=[])
+
+    def get_tables_and_schemas(self):
```

Comment:
Everything looks good so far and the integration test passed for me.

DC-1001 should be done and merged by end of the day which has the permanent location for the FitBit cleaning rules, all the schemas, and the updated `clean_cdr.py` module with a new FitBit cleaning rule stage, should we wait for that to be merged and make the changes or go ahead as is and create a ticket to make those changes later?

Reply:
@dcampbell-vumc  I will wait for your ticket to be done and merged since it is almost ready.  Then we can update this one.  I just had to place holder to drive  development forward.  I prefer to have the schemas split out rather than defined in the classes.